src/components/AddItems.js

```diff
@@ -90,67 +91,76 @@ const AddItemsToList = () => {
 
   return (
     <div>
-      <div>
-        <form onSubmit={submitShoppingListItemHandler}>
-          <h2>Add Item to List</h2>
-          {shoppingListItemNameExists ? (
-            <p>
-              {`You have ${normalizeString(
-                shoppingListItemName,
-              )} in your shopping list already`}
-            </p>
-          ) : null}
-          <div>
-            <label>
-              Name
-              <input
-                type="text"
-                value={shoppingListItemName}
-                onChange={shoppingListItemNameHandler}
-              />
-            </label>
-          </div>
-          <fieldset>
-            How soon are you likely to buy it again?
+      <div className="max-h-screen flex flex-col box-border items-center">
+        <header className="bg-green-400 w-full fixed text-center">
+          <h2 className="pt-6 pb-16 text-4xl font-thin text-gray-100">
+            Add Item to List
+          </h2>
+          <span className="text-white top-0 right-0 absolute sm:mt-4 sm:mr-4">
+            <HomeIcon />
+          </span>
+        </header>
```

Comment:
I really really love the header 🚀 I was wondering if having the `HomeIcon` on the right side could be more intuitive? 

I have one more suggestion: When the screen size is less than 640px, the home icon moves up in the header and as you decrease the screen size to 340px, it overlaps with the heading text. 
Do you think having the icon centered below the heading text(`Your Shopping List`) below 640px might look better? (Of course, this is my observation and just a suggestion). ☺️

Reply:
Oh really? I tried with firefox and google, but the icon is always above the text, even with 300px. The icon is set with top0 and right0 when it is less than 640 to not overlap with the header. but it has a margin when it is more than 640. 
But yes, I am still not sure how to go with this Icon for myself. I will try different things out this week :)

Reply:
Yep, this is what I get below 350px. It is a tiny detail and you can totally ignore it.
One idea I have is to maybe have the icon always centered below the header or on top of the header? 💡
I'm assuming your intention is to have the app logo here instead of home? I'm guessing this because the home icon does not have any purpose other than the design factor? Thanks Julia 😊

<img width="1440" alt="Screen Shot 2021-02-21 at 3 59 34 PM" src="https://user-images.githubusercontent.com/45564632/108624355-78367400-7412-11eb-8038-4350306e6fa9.png">

Reply:
Oh, no, we are going to add a function to be able to go back to the home screen. And in case we do not add the function, the icon will be removed completely. 😊

Reply:
yep, that makes sense. One detail to take into consideration - Currently app is built so that when users have tokens, they cannot access the Home screen.
If you want to add above mentioned functionality, you would need to modify the restriction part as well.